programmers/난이도별/level01.핸드폰_번호_가리기/HyeonJeong.py

```diff
@@ -0,0 +1,5 @@
+def solution(phone_number):
+    answer = ""
+    for i, c in enumerate(phone_number):
+        answer += "*" if i < len(phone_number) - 4 else c
+    return answer
```

Comment:
그리고 굳이 반복문을 사용하지 않아도 한 줄이면 풀이할 수 있습니다!
```suggestion
    return len(phone_number[:-4]) * "*" + phone_number[-4:]
```

혹은

```suggestion
    return (len(phone_number) - 4) * "*" + phone_number[-4:]
```

이렇게요!